release-notes/schemas/dotnet-patch-releases-index.json

```diff
@@ -0,0 +1,50 @@
+{
+  "description": "A set of product releases with high-level information, like latest version.",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "channel-version": {
+      "type": "string"
+    },
+    "releases": {
+      "type": "array",
+      "items": {
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "release-version": {
+            "description": "Version of release.",
```

Comment:
Maybe be more specific about what the version is (it's the "marketing" or "brand" version, right?)